docs/modules/ROOT/pages/cops_style.adoc

```diff
@@ -9985,8 +9985,8 @@ Checks for usage of comparison operators (`==`,
 These can be replaced by their respective predicate methods.
 This cop can also be configured to do the reverse.
 
-This cop can be customized allowed methods with `AllowedMethods`.
-By default, there are no methods to allowed.
+This cop's allowed methods can be customized with `AllowedMethods`.
```

Comment:
This file is auto-generated, you should change it in the cops source instead: https://github.com/rubocop/rubocop/blob/a6927ace7f96a2125d13e9f711b43be899823f43/lib/rubocop/cop/style/numeric_predicate.rb#L11-L12

There are also two other cops with the same wording if I search for "This cop can be customized allowed methods with", would you mind changing these as well? Thanks!

Reply:
@Earlopain fixed :)

I guess I should double check the PRs that I open on Monday morning 😅 😆. Thanks a lot for your feedback!